cldfbench_nominalperson_cldf.py

```diff
@@ -22,26 +23,31 @@ def cmd_download(self, args):
         pass
 
     def cmd_makecldf(self, args):
-        """
-        Convert the raw data to a CLDF dataset.
-
-        >>> args.writer.objects['LanguageTable'].append(...)
-        """
-        
-        with Writer(cldf_spec) as writer:
-            for row in self.raw_dir.read_csv(
-                'examples.csv',
-                dicts=True, 
-            ):
-                writer.objects['ExampleTable'].append({
+        # Extend the schema:
+        args.writer.cldf.add_component('LanguageTable')
+        args.writer.cldf.add_component('ExampleTable')
+
+        for lid, rows in itertools.groupby(
```

Comment:
We goup by Language_ID in order to easily create a language object first, and examples then.